### PR TITLE
Create edificecms.com.website.test.json

### PR DESCRIPTION
**Pull request by sabariskadits**
Add edificecms.com website test template.

### CODE REVIEW

**Comment by sabariskadits** [comment]
Merge my branch codes

File: edificecms.com.website.test.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "providerId": "edificecms.com",
+  "providerName": "edificecms.com",
+  "serviceId": "website.test",
+  "serviceName": "edificecmsdns",
+  "version": 1,
+  "logoUrl": "https://beta.edificecms.com/builder/elements/images/uploads/project576/17089491464566.png",
+  "description": "Configure DNS records using EdificeCMS",
+  "variableDescription": "IP address for the domain A, CNAME. Domain for TXT, SPF, MX",
```
Comment (pawel-kow) [inline edificecms.com.website.test.json:9]: Pls check the variable description.

File: edificecms.com.website.test.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "providerId": "edificecms.com",
+  "providerName": "edificecms.com",
+  "serviceId": "website.test",
+  "serviceName": "edificecmsdns",
+  "version": 1,
+  "logoUrl": "https://beta.edificecms.com/builder/elements/images/uploads/project576/17089491464566.png",
+  "description": "Configure DNS records using EdificeCMS",
```
Comment (pawel-kow) [inline edificecms.com.website.test.json:8]: Add signing if you want to have good adoption with DNS providers.

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: edificecms.com.website.test.json (hunk 1 above)
Comment (sabariskadits) [reply edificecms.com.website.test.json]: We can changed descriptions

**Review by sabariskadits — COMMENTED** [review]
We can changed descriptions

File: edificecms.com.website.test.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "providerId": "edificecms.com",
+  "providerName": "edificecms.com",
+  "serviceId": "website.test",
+  "serviceName": "edificecmsdns",
+  "version": 1,
+  "logoUrl": "https://beta.edificecms.com/builder/elements/images/uploads/project576/17089491464566.png",
+  "description": "Enables web hosting for edificecms domains.",
+  "variableDescription": "IP address for the domain A",
```
Comment (pawel-kow) [inline edificecms.com.website.test.json:9]: I mean there are generally no variables so the description is not needed in fact.

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: edificecms.com.website.test.json (hunk 3 above)
Comment (sabariskadits) [reply edificecms.com.website.test.json]: Variable description changed blank content

**Review by pawel-kow — APPROVED** [review]
Lgtm

**Pull request merged**